go-controller/pkg/config/config.go

```diff
@@ -548,6 +549,11 @@ var K8sFlags = []cli.Flag{
 		Usage:       "specify the ovnkube pod IP.",
 		Destination: &cliConfig.Kubernetes.PodIP,
 	},
+	cli.StringFlag{
+		Name:        "no-hostsubnet-nodes",
+		Usage:       "Specify a label for nodes that will manage their own hostsubnets",
+		Destination: &cliConfig.Kubernetes.NoHostSubnetNodes,
```

Comment:
@JacobTanenbaum per above this should get put into the raw variant.

Reply:
fixed